tensorboard/plugins/interactive_inference/witwidget/notebook/colab/wit.py

```diff
@@ -12,4 +12,319 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO(jameswex): Colab implementation will be in separate PR.
+import base64
+import json
+import tensorflow as tf
+from IPython import display
+from google.colab import output
+from google.protobuf import json_format
+from tensorboard.plugins.interactive_inference.utils import inference_utils
+
+# Python functions for requests from javascript.
+def infer_examples():
+  WitWidget.widget.infer()
+output.register_callback('notebook.InferExamples', infer_examples)
+
```

Comment:
I think its suggested to have two spaces between module functions.

Reply:
done